Python/bytecodes.c

```diff
@@ -657,8 +657,7 @@ dummy_func(
             STACK_SHRINK(2);
             new_frame->localsplus[0] = container;
             new_frame->localsplus[1] = sub;
-            assert(1 + INLINE_CACHE_ENTRIES_BINARY_SUBSCR == next_instr - this_instr);
-            frame->return_offset = 1 + INLINE_CACHE_ENTRIES_BINARY_SUBSCR;
+            frame->return_offset = next_instr - this_instr;
```

Comment:
This probably needs a cast to uint16_t

Reply:
It's guaranteed to be small. So, I think it should be OK. Let's see if MSVC complains.

Reply:
It does, indeed, complain